cmd/server-mux.go

```diff
@@ -420,21 +451,110 @@ func (m *ServerMux) ListenAndServe(certFile, keyFile string) (err error) {
 		MinVersion: tls.VersionTLS12,
 	} // Always instantiate.
 
-	if tlsEnabled {
-		// Configure TLS in the server
-		if config.NextProtos == nil {
-			config.NextProtos = []string{"http/1.1", "h2"}
+	// Configure TLS in the server
+	if config.NextProtos == nil {
+		config.NextProtos = []string{"http/1.1", "h2"}
+	}
+	config.Certificates = make([]tls.Certificate, 1)
+
+	config.Certificates[0], tlsErr = tls.X509KeyPair(m.certPEM, m.keyPEM)
+	if tlsErr != nil {
+		return nil, false, false, tlsErr
+	}
+
+	// Fetch for OCSP response
+	ocspBytes, _, ocspErr := ocsp.GetOCSPForCert(m.certPEM)
```

Comment:
> Server running Revoked certificate should be lo logged properly.

Done



> Server to server communication should not continue to work.
> Even server could return one of 5xx code and stop doing real work for requests.

I think that, if we are not providing a new config option to bypass ocsp check, server/cluster needs to continue to work normally even if the certificate is revoked rather than stopping everyting. Some clients, like Chrome, already don't rely on OCSP at all and I think we don't need to be too much rigorous.